core/include/reboot.h

```diff
@@ -28,7 +30,10 @@
  *
  * This function is used by core_panic() when the DEVELHELP macro is not defined.
  */
-void reboot(void);
+static inline void reboot(void)
+{
+    pm_reboot();
+}
```

Comment:
question: why don't we just throw out this file completely and use `pm_reboot` directly in `core_panic`?